pom.xml

```diff
@@ -224,4 +224,17 @@
     </plugins>
   </reporting>
 
+  <profiles>
+    <profile>
+      <!-- JDK 9+ has the "release" option to ensure the generated bytecode is
+       compatible with Java 8. -->
+      <id>compiler-release-8</id>
```

Comment:
This profile may be appropriate to be defined in the google-cloud-shared-config.

Reply:
Added todo: https://github.com/googleapis/java-shared-config/issues/477